sql-bricks.js

```diff
@@ -41,7 +41,7 @@
 
     var str = this.str;
     if (!opts)
-      opts = default_opts;
+      opts = _.extend({}, default_opts);

```

Comment:
@vitalif: I think `_.clone(default_opts)` would be a littler clearer than `_.extend({}, default_opts);` (also below in `Group.prototype.toString`).